theme/js/components/search/search.vue

```diff
@@ -0,0 +1,293 @@
+      {{ $t("@@Jeux de données") }}<sup>{{ total_results || 0 }}</sup>
+    </h1>
+    <a :href="reuseUrl" title="" class="nav-link fs-sm mt-lg-sm hidden-md">
+      {{ $t("@@Rechercher dans les réutilisations") }}
+    </a>
+  </div>
+  <section class="search-filters p-md-md" :class="extendedForm && 'active'">
+    <h4 class="mt-md mt-md-0 mb-xs mb-md-md fs-sm">
+      {{ $t("@@Critères de recherche") }}
+    </h4>
+    <div class="filters-wrapper p-sm p-md-0">
+      <div class="row justify-between align-items-center">
+        <div class="col-3 col-lg-6 col-md-12">
+          <Suggestor
+            :placeholder="$t('@@Organisations')"
+            :searchPlaceholder="$t('@@Chercher une organisation...')"
+            listUrl="/organizations/"
+            suggestUrl="/organizations/suggest/"
+            entityUrl="/organizations/"
+            :values="facets.organization"
+            :onChange="handleSuggestorChange('organization')"
+          />
+        </div>
+        <div class="col-3 col-lg-6 col-md-12 mt-md-md">
+          <Suggestor
+            :placeholder="$t('@@Mots clés')"
+            :searchPlaceholder="$t('@@Chercher un mot clé...')"
+            suggestUrl="/tags/suggest/"
+            :values="facets.keywords"
+            :onChange="handleSuggestorChange('keywords')"
+          />
+        </div>
+        <div class="col-3 col-lg-5 col-md-12 mt-lg-md">
+          <Suggestor
+            :placeholder="$t('@@Licenses')"
+            :searchPlaceholder="$t('@@Chercher une license...')"
+            listUrl="/datasets/licenses/"
+            :values="facets.license"
+            :onChange="handleSuggestorChange('license')"
+          />
+        </div>
+        <div class="col-2 col-lg-5 col-md-12 mt-lg-md">
+          <Suggestor
+            :placeholder="$t('@@Formats')"
+            :searchPlaceholder="$t('@@Chercher un format...')"
+            suggestUrl="/datasets/suggest/formats/"
+            :values="facets.format"
+            :onChange="handleSuggestorChange('format')"
+          />
+        </div>
+        <div
+          class="col-1 col-lg-2 hidden-md text-align-center mt-lg-md text-align-right-lg"
+        >
+          <a
+            class="btn-secondary btn-secondary-grey-200 px-md"
+            @click="extendedForm = !extendedForm"
+          >
+            <span v-if="!extendedForm">&#8230;</span>
+            <span v-else>X</span>
+          </a>
+        </div>
+      </div>
+      <div v-if="extendedForm" class="row mt-sm align-items-center">
+        <div class="col-5 col-lg-6 col-md-12 row-inline">
+          <Rangepicker
+            :value="facets.temporal_coverage"
+            :onChange="handleSuggestorChange('temporal_coverage')"
+          />
+        </div>
+        <div class="col-3 col-md-12 mt-md-md">
+          <Suggestor
+            :placeholder="$t('@@Zone géographique')"
+            :searchPlaceholder="$t('@@Chercher une zone...')"
+            suggestUrl="/spatial/zones/suggest"
```

Comment:
```suggestion
            suggestUrl="/spatial/zones/suggest/"
```

BTW, we never discussed abstracting the API in a service or smtg?